src/Nox.ClientApi.Tests/Tests/Controllers/StoresControllerTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentAssertions;
+using ClientApi.Application.Dto;
+
+namespace Nox.ClientApi.Tests.Tests.Controllers
+{
+    [Collection("Sequential")]
+    public class StoresControllerTests : NoxIntgrationTestBase
+    {
+        private const string StoresControllerName = "api/stores";
+
+        public StoresControllerTests(NoxTestApplicationFactory<StartupFixture> factory) : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task Post_ReturnsNuidId()
+        {
+            // Arrange
+            string name = "MySpecialName";
+            uint expectedId = 2519540169u;
+
+            var createDto = new StoreCreateDto
+            {
+                Name = name,
+                // TODO make email mandatory
+                //EmailAddress = new EmailAddressUpdateDto()
```

Comment:
When TODO will be done? Do we need it?

Reply:
I'm not aware of this. 
I left comments as they used to be, my work was to change replace controlerr's methods invocation to httpClient request.